scripts/ci/gitlab/pipeline/build.yml

```diff
@@ -169,7 +169,8 @@ build-implementers-guide:
     - .test-refs
     - .collect-artifacts-short
   script:
-    - cargo install mdbook mdbook-mermaid mdbook-linkcheck
+    - apt-get update && apt-get install -y graphviz
```

Comment:
I think you need `apt-get -y update; apt-get -y graphviz` to be non interactive. See higher in the file.

Reply:
```suggestion
    - apt-get -y update; apt-get install -y graphviz
```